Whisky.xcodeproj/project.pbxproj

```diff
@@ -66,6 +66,9 @@
 		6EFDF6632AAE2E1700EF622F /* DataExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6E26058929D9282000DDD788 /* DataExtensions.swift */; };
 		6EFDF6642AAE2E1B00EF622F /* URLExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6E5D41792A39021B0038ABEA /* URLExtensions.swift */; };
 		6EFDF6662AAE303300EF622F /* Icons.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 6EFDF6652AAE303300EF622F /* Icons.xcassets */; };
+		9A3078F42ACC06CC00419D85 /* Program.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6E355E5F29D7D8BD002D83BE /* Program.swift */; };
```

Comment:
What are these changes to the project file for?

Reply:
I imported some of the whisky libraries from the whisky app to be able to run it in Whisky cmd (Program.swift and Wine.swift)

Reply:
I'm hoping we can also move the CLI to a Swift Package some time soon since SPM supports executable package types. This would prevent this kind of importation. I personally think it's best that whatever dependencies be moved to `WhiskyKit` if possible.